src/github.ts

```diff
@@ -312,31 +320,36 @@ export class GitHub {
     maxLabels = 16,
     retries = 0
   ): Promise<CommitsResponse> {
+    const baseBranch = await this.getDefaultBranch();
```

Comment:
Should this be a method argument where the caller can provide the base branch name?

Reply:
Yeah, I saw this could go either way... what we had before was that methods had to supply options.defaultBranch each time, and then it got cached, so the future invocations ignored the parameter. Which seems a bit deceptive.

I'm fine with going the other way, too, just make it have to be passed each time and caching the result value by key or something.

Reply:
Oh, I missed your last comment - my intent is that this first pass would just allow someone to run it from the command line on a onesies-basis, but I'm intending to follow it up with one that will read a list of branches from the `.release-please.yaml` file in a repo. So I'm good with assuming that's where things need to go, one way or another.